int16 uisteerangle
int16 realstrtorque
float32 uispeed
int16 espaccel
uint8 shiftlvlposition
uint8 currentdrvmode
uint8 sysstatus
uint8 vehiclesysfault
int16 batteryvoltage
int8  reserve1
float32 reserve2
uint8  reserve3
uint8  reserve4
bool isvalid

